c2cgeoportal/scaffolds/create/vars_+package+.yaml_tmpl

```diff
@@ -115,6 +115,10 @@ vars:
             The GeoMapfish team
         smtp_server: smtp.example.com
 
+    headers:
+        mapserver:
+            cors: *
```

Comment:
Use from the Cartoriviera portal with the WMSBrowser the WMS from the Vaud
> portal without the OGSProxy...
>


So this is a specific case. I just think that "*" should not be the default
value.


-- 
Eric Lemoine

Camptocamp France SAS
Savoie Technolac, BP 352
73377 Le Bourget du Lac, Cedex

Tel : +33 (0)4 58 48 20 34
Mail : eric.lemoine@camptocamp.com
http://www.camptocamp.com